frontend/src/types.ts

```diff
@@ -666,5 +666,6 @@ export type IntegrationAppStatus = {
   isInstalled: boolean;
   isEnabled: boolean;
   canInstall: boolean;
+  variablesValidationStatus: string;
```

Comment:
Yeah I wasn't sure about this. But the variable name specifies status, and the validation come directly from the cr yaml `.status` fields. This means the frontend needs to handle and know the different error codes that can happen in kubernetes.

There are more than 2 states it can be from testing. "Unknown" for example. 

There also is the option for the backend to handle the parse kubernetes validation states and return if it is valid as a boolean.

Reply:
This CR is created by the backend, we should be able to know the status for this condition field... 🤔

Reply:
@andrewballantyne @emilys314 As I tested, the `variablesValidationStatus ` could be ''True'', ''False'', `'Unknown'`, `''`, maybe ENUM is better?